platform/core/src/services/HangingProtocolService/HangingProtocolService.ts

```diff
@@ -264,23 +299,40 @@ class HangingProtocolService {
     this.activeProtocolIds = [...protocolId];
   }
 
+  /**
+   * Sets the active study.
+   * This is the study that the hanging protocol will consider active and
+   * may or may not be the study that is being shown by the protocol currently,
+   * for example, a prior view hanging protocol will NOT show the active study
+   * specifically, but will show another study instead.
+   */
+  public setActiveStudyUID(activeStudyUID: string): void {
```

Comment:
Should this be added to public API documentation?

Reply:
It is in the public API documentation - that is generated from the javadocs annotation, so it is only other/higher level behaviour that needs to be included in the md file.